src/actinia_core/core/stac_importer_interface.py

```diff
@@ -95,37 +93,36 @@ def _apply_filter(stac_root_search, stac_name, interval, bbox, filter):
 
         full_filtered_result = stac_search.json()
 
-        if "features" in full_filtered_result:
+        if "features" in full_filtered_result\
```

Comment:
whitespace before `\`, check with black or flake8
```suggestion
        if "features" in full_filtered_result \
```